base/models.py

```diff
@@ -33,14 +34,29 @@ class GalaxyInstance(models.Model):
     # Aggregate Job Data
     jobs_run = models.IntegerField(default=0)
 
+class ConvertUnixTimestamp(models.DateTimeField):
```

Comment:
interesting way to do this @elenimijalis, I must not have understood quite what you were saying when you talked about doing it "in a class". This looks like it should work OK, just a different way than I would've implemented it.